src/views/Timeline.vue

```diff
@@ -4,7 +4,7 @@
     justify="space-around"
   >
     <v-col
-      v-if="!isEdgeConnected"
+      v-if="connectionStatus !== 'PEER_CONNECTED'"
```

Comment:
Instead of mutating the `edgeConnected` value directly in the store, i am copying the value from the store so i can simply access it as a data value in the component.

The goal for the tests is to detach the component or anything that relates to testing the component from the central VueX store

Reply:
Doesn't sound convincing. Let's think about encapsulating the tested code with a known pre-test state, running the test and verifying output against expected results. 

In this case, think about setting the value of isEdgeConnected prior to running a test and then verifying output against expected results.